src/nupic/regions/ScalarSensor.hpp

```diff
@@ -67,8 +67,7 @@ class ScalarSensor : public RegionImpl, Serializable {
   virtual std::string executeCommand(const std::vector<std::string> &args,
                                      Int64 index) override;
 
-  virtual size_t
-  getNodeOutputElementCount(const std::string &outputName) const override;
```

Comment:
API compat ok?

Reply:
> removed getNodeOutputElementCount(const std::string &outputName) const override;
API compat ok?

yes, the base class handles it.